pkg/utils/util.go

```diff
@@ -287,3 +292,27 @@ func AppendUniqNoneEmptyString(slice []string, element string) []string {
 	}
 	return append(slice, element) // Append the element
 }
+
+// GetJiraClient creates a jira client that connects to
+// https://issues.redhat.com. To work, the jiraToken needs to be set in the
+// config
+func GetJiraClient() (*jira.Client, error) {
+	var jiratoken string
+
+	if !viper.IsSet(JiraTokenConfigKey) {
+		jiratoken = viper.GetString(JiraTokenConfigKey)
+	}
+
+	if os.Getenv("JIRA_API_TOKEN") != "" {
+		jiratoken = os.Getenv("JIRA_API_TOKEN")
+	}
+
+	if jiratoken == "" {
+		return nil, fmt.Errorf("JIRA token is not defined")
+	}
+
+	tp := jira.PATAuthTransport{
+		Token: jiratoken,
+	}
+	return jira.NewClient(tp.Client(), JiraBaseURL)
+}
```

Comment:
This function nicely matches to the backplane config module rather than the util func 
https://github.com/openshift/backplane-cli/blob/main/pkg/cli/config/config.go

But we should create a wrapper JIRA interface for unit test purposes 
something like pagerduty client 
https://github.com/openshift/backplane-cli/blob/main/pkg/pagerduty/client.go

Reply:
Sure I can move it there. However this `GetJiraClient` won't use any of the internal members of `BackplaneConfiguration`. So it won't be an object method there.

Ok to add some interface layer for the sake of mocking.
Will check if one already exists in the JIRA module.